book/pipelines.md

```diff
@@ -83,3 +83,53 @@ And the pipeline:
 ```
 
 Are one and the same.
+
+And if you want to see the original form, you can use [`to text`](commands/to_text.md):
```

Comment:
Hmm. `to text` converts structured data into text in an opinionated way; it's not exactly "the original form"

Reply:
Thanks for correction